src/source/rodash/isArrayLike.bs

```diff
@@ -1,15 +1,19 @@
 namespace rodash
-  ' Checks if value is array-like. A value is considered array-like if it is an array, a string, or an node with children.
+  ' Checks if value is array-like. A value is considered array-like if it is an array, roList, roByteArray, or roXMLList
   ' @since 0.0.21
   ' @category Lang
   ' @param {Dynamic} value The variable to be checked
   ' @returns {Boolean} Results of the check
   ' @example
   ' rodash.isArrayLike([]) // => true
+  ' rodash.isArrayLike(CreateObject("roList")) // => true
+  ' rodash.isArrayLike(CreateObject("roByteArray")) // => true
+  ' rodash.isArrayLike(CreateObject("roXMLList")) // => true
   ' rodash.isArrayLike({}) // => false
+  ' rodash.isArrayLike("example string") // => false
   @inline
   function isArrayLike(value as Dynamic) as Boolean
-    return rodash.isArray(value) OR rodash.isString(value) OR rodash.isNodeWithChildren(value)
+    return rodash.isNotInvalid(value) AND getInterface(value, "ifArrayGet") <> Invalid
```

Comment:
Nice. Glad we caught these edge cases for things that should have been treated as arrays but weren't.